<ui version="4.0" >
 <class>ImportWidgetClass</class>
 <widget class="QWidget" name="ImportWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ImportWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>411</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title" >
    <string>测量数据导入</string>
   </property>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>导入项目:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameLineEdit" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>140</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>140</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>浏览...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="importButton" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>开始导入</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>180</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value" >
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="dataComboBox" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>30</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>410</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="textFormat" >
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="countLabel" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>70</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>100</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>格式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="excelRadioButton" >
       <property name="text" >
        <string>Excel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="csvRadioButton" >
       <property name="text" >
        <string>CSV</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="itemCountLabel" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>220</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>470</x>
     <y>20</y>
     <width>421</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>三维点云模型导入</string>
   </property>
   <widget class="QProgressBar" name="modelProgressBar" >
    <property name="geometry" >
     <rect>
      <x>60</x>
      <y>70</y>
      <width>341</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value" >
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="importModelBtn" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>30</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>导入模型...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>290</y>
     <width>411</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>照片导入</string>
   </property>
   <widget class="QPushButton" name="imPhotoBtn" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>30</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>导入照片...</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="photoProgressBar" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>70</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value" >
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>630</x>
     <y>150</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>生成统计数据</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="statisticProgressBar" >
   <property name="geometry" >
    <rect>
     <x>500</x>
     <y>200</y>
     <width>441</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value" >
    <number>0</number>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget" >
   <property name="geometry" >
    <rect>
     <x>500</x>
     <y>260</y>
     <width>411</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="VisiblePushButton" >
   <property name="geometry" >
    <rect>
     <x>500</x>
     <y>227</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>显示详细信息</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
